metrics-exporter-prometheus/src/distribution.rs

```diff
@@ -95,7 +108,10 @@ impl DistributionBuilder {
             return Distribution::new_histogram(buckets);
         }
 
-        Distribution::new_summary(self.quantiles.clone())
+        let b_duration = self.bucket_duration.map_or(Duration::from_secs(20), |d| d);
+        let b_count = self.bucket_count.map_or(3, |c| c);
```

Comment:
Since we're now referencing these default in two places -- here, and `RollingSummary::default` -- we should lift them both to constants at the top of the file... like `DEFAULT_SUMMARY_BUCKET_COUNT`, etc.

(We can't construct the `NonZeroU32` in a const fashion, so just the `u32` itself will be fine for now.)